RecoLocalTracker/SiPixelRecHits/src/SiPixelGenError.cc

```diff
@@ -308,6 +319,11 @@ bool SiPixelGenError::pushfile(const SiPixelGenErrorDBObject& dbobject,
       
       // next, layout the 1-d/2-d structures needed to store GenError
       
+      // &&&  Who is going to delete these?  Are we leaking memory?
```

Comment:
i have the same question... perhaps you should just make these vectors. Would be much simpler.